Vagrantfile

```diff
@@ -55,7 +55,7 @@ Vagrant.configure("2") do |config|
   #   vb.gui = true
   #
   #   # Customize the amount of memory on the VM:
-  vb.memory = "1024"
+    vb.memory = "1024"
```

Comment:
Is that intentional?
The indentation looks different from the other configs

Reply:
Yes, following @ifireball 's comment on this on the previous PR.

Reply:
OK.
Looks fine to me (I also cloned it and ran it on my machine, looks great).
@ifireball do you have any comments, or can we merge it?